LICENSE

```diff
@@ -307,11 +286,6 @@ For details see http://jquery.org/license
 
 ++++++++++++++++++++++++++++++
 
-Apache Trafodion uses asciidocs to create its documentation which is licensed
-under MIT 
-
-For details see http://asciidoctor.org/
```

Comment:
Just to confirm - an original comment we received for our first release attempt (that did include docs) was that we needed to add asciidocs to our license file.  When we released 1.3.0, we did not include docs so we then received a comment that this text was not needed.  Are we going to include docs in our source tree?  If so, will we need this text?